src/components/overlay/overlay.vue

```diff
@@ -8,15 +8,15 @@
             <div class="overlay--root" v-if="overlay.isOpen" :key="overlay.timestamp">
                 <div class="overlay--backdrop"></div>
                 <div class="overlay--display">
-                    <div class="overlay--wrap-outer">
+                    <div class="overlay--wrap-outer" :style="overlayScrollLockStyles">
                         <div class="overlay--wrap-inner">
                             <div class="overlay--backdrop-click-area"
                                     @click.prevent="closeOverlay({ id })"></div>
                             <div class="overlay--container">
                                 <div class="overlay--revealer">
-                                    <cmp-modal :facets="overlay.facets">
+                                    <UiModal :facets="overlay.facets">
```

Comment:
why did we change from dashcase to camel case for the components?

Reply:
copy waste